config/main.py

```diff
@@ -238,6 +238,31 @@ def _change_bgp_session_status(ipaddr_or_hostname, status, verbose):
     for ip_addr in ip_addrs:
         _change_bgp_session_status_by_addr(ip_addr, status, verbose)
 
+def _validate_bgp_neighbor_ip(neighbor):
```

Comment:
Now that the function can take either an IP address or hostname, I believe this function name and parameter name should be changed to be more clear. Maybe something like:

`def _validate_bgp_neighbor(ipaddr_or_hostname):`

Reply:
changed as suggested in next iteration

Reply:
You changed the parameter name, but not the function name. Now that the function can take either an IP address or hostname, you should remove "ip" from the function name, as in my example above, `_validate_bgp_neighbor()`

Reply:
Hi @jleveque, 
Renamed the method name. 
I can see the test failed but it is not related to my change.
Kindly help in retest the build